irods/message/__init__.py

```diff
@@ -150,19 +150,24 @@ def ET(xml_type = 0, server_version = None):
     UNICODE = str
 
 
+import fcntl, os
+
+# Necessary for older python (<3.7):
+_socket_is_blocking = (lambda self: 0 == fcntl.fcntl(self.fileno(), fcntl.F_GETFL) & os.O_NONBLOCK)
```

Comment:
Would it make sense to sniff the python version here or use a try-except like on 145-150?

Reply:
This will work for old Python as well as new.  I guess to make things clearer for myself later, I could restore the `getblocking()` call and try/except or... perhaps more explicitly... test sys.version_info >= (3,7). I'm agnostic which, except it involves more effort.

Reply:
You can leave it as-is. Just like to use the boring things when they're available :p